components/digid-button/css/story-template.jsx

```diff
@@ -16,11 +16,11 @@ export const argTypes = {
     control: 'text',
   },
 };
+export const defaultArgs = { label: '', textContent: '' };
+export const exampleArgs = { label: 'Log in using DigiD', textContent: 'Log in' };
 
-export const defaultArgs = { label: 'Log in using DigiD', textContent: 'Log in' };
-
-export const DigidButton = ({ label = '', textContent = '' }) => (
-  <div class="utrecht-digid-button">
+export const DigidButton = ({ label = defaultArgs.label, textContent = defaultArgs.textContent }) => (
+  <div className="utrecht-digid-button">
     <utrecht-digid-logo className="utrecht-digid-button__logo"></utrecht-digid-logo>
```

Comment:
It appears that `className` on custom elements is not recognized by the HTML serializer, and is rendered as `className` instead of `class` in the code snippet :(